### ISSUE TITLE
Slash Command: /manage

### ISSUE THREAD

**Issue by StephanWells** [Raid-Training-Initiative/RTIBot#345]
**Details:**
* `/manage trainingrequest` is a subcommand group.
  - `/manage trainingrequest add <member> <wings> <comment>` adds a training request for the given `<member>`, with the `<wings>` being a comma-separated list of wings (use `8` for EoD strikes) and `<comment>` being a comment that shows up in the training request.
  - `/manage trainingrequest remove <user>` removes any training request of a given `<member>`
* `/manage application` is a subcommand group.
  - `/manage application add <user>` starts the application process dialogue for the given `<user>`.
* `/manage member` is a subcommand group.
 - `/manage member ban <member> [duration]` bans the given `<member>`, optionally with an unban scheduled in the specified `[duration]`.
    - `[duration]` can be written in plain English (e.g. "2 weeks", "1 year", "25 seconds", etc).
    - A banned member should have their banned status changed to `true`, their Discord member role removed, and kicked from all raids they are a part of. 
  - `/manage member unban <member>` unbans the given `<member>` if they are already banned.
  - `/manage member edit <member> <ign>` sets a new `<ign>` for the given `<member>`.
  - `/manage member remove <member>` removes a given `<member>` from our database.

**Current functionality:**
* `TrainingRequestAddCommand`
* `TrainingRequestRemoveCommand`
* `GuildApplicationAddCommand`
* `MemberBanCommand`
* `MemberEditCommand`
* `MemberRemoveCommand`
* `MemberUnbanCommand`

**Notes:**
* Ensure that there's validation on the `<member>` parameters (i.e. it accepts only Discord users).
* Validation on the `<wings>` parameter and IGNs via regex would be nice.
* `/manage member remove` is not needed if there's a maintenance task that removes people without the Discord member role from our database.
* No autocomplete needed.
* Error messages should be ephemeral.

**Comment by Akhronox** [comment]
Ready in branch feature/slash-migration-autocomplete. To be fully tested, old command files not deleted yet.
